project/config/lgbcni/config/core.entity_form_display.paragraph.faq_accordion.default.yml

```diff
@@ -6,31 +6,15 @@ dependencies:
     - field.field.paragraph.faq_accordion.field_answer
     - field.field.paragraph.faq_accordion.field_question
     - paragraphs.paragraphs_type.faq_accordion
-  module:
-    - text
 _core:
   default_config_hash: 1SEm1W1dc0VrhYJ8MPthdyrWiP-n44xFZCD2TWvOEug
 id: paragraph.faq_accordion.default
 targetEntityType: paragraph
 bundle: faq_accordion
 mode: default
-content:
-  field_answer:
-    weight: 1
-    settings:
-      rows: 5
-      placeholder: ''
-    third_party_settings: {  }
-    type: text_textarea
-    region: content
-  field_question:
-    weight: 0
-    settings:
-      size: 60
-      placeholder: ''
-    third_party_settings: {  }
-    type: string_textfield
-    region: content
+content: {  }
 hidden:
   created: true
+  field_answer: true
```

Comment:
Looks like you are not displaying the question and answer fields on the FAQ form?

Reply:
Good find, thats it fixed and updated